cmd/govim/testdata/scenario_default/quickfix_config.txt

```diff
@@ -3,20 +3,21 @@
 # Default behaviour is quickfix autodiagnostics & sign placment enabled
 vim ex 'e main.go'
 errlogmatch 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/main.go
+errlogmatch -peek 'sendJSONMsg: \[0,\[\d+,"call","s:batchCall",\[\["call","s:mustNothing","setqflist"'
```

Comment:
Do we need `-peek` here? IIRC errlogmatch doesn't consume any lines after a matched line anymore.
If we do need it, shouldn't it be added to the PublishDiagnostics match above as well?

Reply:
I don't think we do, no. Reason being, we know the diagnostics notification will come from `gopls` first. Then, without attaching too much to the current implementation (which is a bit of a moot point because that's exactly what we are doing) we then say the quickfix update and signs update can happen in either order. In reality, they will always happen in the same order, but the `-peek` is really to loosen that a bit. But all of these assertions are so tightly coupled to the implementation it's really not worth it in some respects.

Reply:
I see, good point 👍